sample-compose/app/src/main/java/co/nimblehq/sample/compose/extension/ViewModelExt.kt

```diff
@@ -3,23 +3,19 @@ package co.nimblehq.sample.compose.extension
 import androidx.activity.ComponentActivity
 import androidx.activity.viewModels
 import androidx.annotation.MainThread
-import androidx.fragment.app.*
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
 import androidx.lifecycle.*
 
 /**
  * PLEASE READ THIS BEFORE IMPLEMENT:
- * Right now, there is no easy way to mock/ fake the viewModel inside the Fragment when applying
- * the 'by viewModels()' Kotlin property delegate from the activity-ktx/ fragment-ktx artifact.
+ * Right now, there is no easy way to mock / fake the viewModel inside the Fragment when applying
```

Comment:
```suggestion
 * Right now, there is no easy way to mock/fake the viewModel inside the Fragment when applying
```
I think the correct grammar is to omit the space before and after the `/`. Confirmed by Grammarly.